lib/backend/model/bgppeer.go

```diff
@@ -183,5 +187,6 @@ type BGPPeer struct {
 	// value as a string in to the backend, because confd templating
 	// converts large uints to float e notation which breaks the BIRD
 	// configuration.
-	ASNum numorstring.ASNumber `json:"as_num,string"`
+	ASNum               numorstring.ASNumber `json:"as_num,string"`
+	KeepOriginalNextHop bool                 `json:"keep_next_hop,omitempty"`
```

Comment:
Probably need conversion logic to fill this field in when converting from the "v3" type to the "model" type.  Confd uses the old "model" types, I believe so we have to convert from the v3 to the model types.

Reply:
Thank you @fasaxc for your feedk back! I am a very nubie here, it was hard to follow the "bread crumb trail", mainly try to understand different api kinds and its connections. I suppose that is there an updating process in course. So ts possible that I did unnecessary things. But I could not see where to convert types and at the end confd assumed a boolean value in the Bird Template. It is possible some strange values (not a boolean) inadvertently be used?